Makefile

```
@@ -6,7 +6,7 @@ install:
 	pip install -e .[flask,falcon,starlette,dev]
 
 test:
-	pip install falcon --upgrade
+	pip install falcon starlette --upgrade
```

```suggestion
	pip install falcon --upgrade
```
The reason I put "falcon" here is that falcon2 is incompatible with falcon3. So I need to run tests for both of them.

I think starlette doesn't have this issue?

---

yes I assumed it's to test multiple Falcon's versions. The problem I stumbled upon was that the tests require Starlette to be installed, otherwise it fails with `ImportError`, so I hijacked this to install starlette as needed.

---

I guess you mean when you run the `make test` in your machine, the "starlette" is required?

Actually, you can run `make install`. So all the required libraries will be installed.

CI:
* make install
* make lint
* make test

---

> Actually, you can run make install

that's fair, I'll revert this then